auth/jwt.go

```diff
@@ -0,0 +1,66 @@
+package auth
+
+import (
+	"github.com/dgrijalva/jwt-go"
+
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"time"
+)
+
+type Jwt struct {
```

Comment:
Can you think of some other name for this structure? It is _not_ a JWT per se. Disclaimer: I know that you simply followed the chat example here, but still..